pkg/reconciler/inmemorychannel/dispatcher/controller.go

```diff
@@ -75,8 +74,12 @@ func NewController(
 
 	inmemorychannelInformer := inmemorychannelinformer.Get(ctx)
 
+	configStore := channel.NewEventDispatcherConfigStore(base.Logger)
+	configStore.WatchConfigs(cmw)
```

Comment:
This is nice! Is the CloudEvent client updated with the new config? You probably need to provide an `onAfterStore` callback.

Reply:
Do we want to 1) update the client immediately when the configmap updates or 2) apply the new config the next time imc `Reconcile` loop runs?
The current implementation does the latter. But I guess we can update immediately as you suggested, if that's the desired behavior.

Reply:
yes we want 1) otherwise it would be too confusing. Maybe the solution is to trigger the reconcile loop when the configmap changes.

Reply:
Great! I added the logic. Check the `resync` func in controller.go.